apps/molgenis-components/src/components/forms/EditModal.vue

```diff
@@ -134,9 +134,9 @@ export default {
       schemaMetaData: null as unknown as ISchemaMetaData,
       client: null as unknown as INewClient,
       errorMessage: "",
-      loaded: true,
+      loaded: false,
       currentPage: 1,
-      useChapters: true,
+      myUseChapters: this.useChapters,
```

Comment:
we might need a better name. Something like 'hasUseChapters' or something?

Reply:
probably good to look for a convention of this, the combo of prop vs server backed state.

Reply:
there is no real convention that i know, vue 2 used to use thisValue to break the reactive chain between prop and reactive mutable value